build.sbt

```diff
@@ -134,6 +135,17 @@ lazy val filter = (project in file("filter")).
     )
   ).dependsOn(dropwizardCommon)
 
+lazy val grpc = (project in file("grpc")).
+  settings(
+    commonSettings,
+    javaSettings,
+    name := "metrics-grpc",
+    libraryDependencies ++= Seq(
+      "io.grpc" % "grpc-services" % Versions.grpc,
```

Comment:
Are both dependencies required? Wouldn't be `grpc-core` sufficient?

Reply:
I was convinced they are but both interceptor interfaces are indeed in to `grpc-core`. Fixed.